pkg/utils/tekton/controller_test.go

```diff
@@ -100,7 +100,7 @@ func TestFindingCosignResults(t *testing.T) {
 
 			client := fake.NewClientBuilder().WithLists(&tags).Build()
 
-			result, err := findCosignResultsForImage("image-registry.openshift-image-registry.svc:5000/test-namespace/test-image", client)
+			result, err := findCosignResultsForImage("image-registry.openshift-image-registry.svc:5000/test-namespace/test-image@sha256-hash", client)
```

Comment:
Hi @lcarva I had to do some updates to unit tests based on changes in your PR https://github.com/redhat-appstudio/e2e-tests/pull/61

Could you please check if it looks ok? Thanks

Reply:
It does. Thank you for fixing this. I was unaware that there were unit tests to the e2e tests! Happy surprise.